identity-core/src/common/timestamp.rs

```diff
@@ -37,10 +37,24 @@ impl Timestamp {
   /// fractional seconds truncated.
   ///
   /// See the [`datetime` DID-core specification](https://www.w3.org/TR/did-core/#production).
+  #[cfg(not(all(target_arch = "wasm32", not(target_os = "wasi"), feature = "wasm")))]
   pub fn now_utc() -> Self {
     Self(truncate_fractional_seconds(OffsetDateTime::now_utc()))
   }
 
+  /// Creates a new `Timestamp` with the current date and time, normalized to UTC+00:00 with
+  /// fractional seconds truncated.
+  ///
+  /// See the [`datetime` DID-core specification](https://www.w3.org/TR/did-core/#production).
+  #[cfg(all(target_arch = "wasm32", not(target_os = "wasi"), feature = "wasm"))]
+  pub fn now_utc() -> Self {
+    let date = js_sys::Date::new_0();
+    let milliseconds_since_unix_epoch: u64 = date.get_time() as u64;
```

Comment:
Nit: You could use [`Date::now`](https://docs.rs/js-sys/latest/js_sys/struct.Date.html#method.now) here.